CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to rpsrtsrs
+
+#### Table Of Contents
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
```

Comment:
Is it possible to create the TOC automatically with GitHub markdown magic or something? If not I'm perfectly fine with this.